includes/polling/os/allworx.inc.php

```diff
@@ -0,0 +1,4 @@
+<?php
+$hardware = $device['sysDescr'];
+$version = snmp_get($device, '1.3.6.1.2.1.27.1.1.4.1', '-OQv', '', '');
```

Comment:
You should add a license header to the files you add.

Can you please convert these two snmp_gets into multi gets - check other files for examples.